extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/security/FormAuthCookiesTestCase.java

```diff
@@ -60,6 +60,7 @@ public class FormAuthCookiesTestCase {
             "quarkus.http.auth.form.cookie-name=laitnederc-sukrauq\n" +
             "quarkus.http.auth.form.cookie-same-site=lax\n" +
             "quarkus.http.auth.form.http-only-cookie=true\n" +
+            "quarkus.http.auth.form.cookie-max-age=PT1S\n" +
```

Comment:
Please set higher value, few seconds, not one. I think it would pass most of the time, but one second is asking for flakiness. This way, your changes won't have impact on existing logic behind the test.

Reply:
We're setting a value server-side then reading it client-side, wouldn't it work regardless of the time it takes for the test to run?

I'll set a higher value for the test anyway, just in case :)

Reply:
> We're setting a value server-side then reading it client-side, wouldn't it work regardless of the time it takes for the test to run?

Cookies are kept and used by the RestAssured for next requests. Wouldn't it rely on RestAssured `CookieFilter` behavior to work/on work?

Reply:
Oh thank you, I completely missed the `CookieFilter`.

Reply:
np